guide/17-geocoding-guides/part1_what_is_geocoding.ipynb

```diff
@@ -0,0 +1,1334 @@
+{
```

Comment:
<p>-&nbsp;Batch geocoding: Geocode place of interest or address queries with a single request using the Geocoding Service. An ArcGIS Developer or ArcGIS Online subscription is required to use the service. It is&nbsp;also known as <code>bulk geocoding</code>, the process of converting a list of addresses or place names to a set of complete addresses with locations. Batch geocoding is commonly used to:&nbsp;convert a number of addresses to complete addresses,&nbsp;find the locations for a list of addresses,&nbsp;perform large batches of geocoding, or&nbsp;geocode addresses that can be saved for future use.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>